src/main.py

```diff
@@ -1,11 +1,61 @@
 import sys
+import os
+from random import randint
+from Board import initBoard, getBoardFromConfig, displayBoard, shoot
+from constants import UNICODE_FOR_A_CHAR
+from Game import boards, getPrimaryBoard, getOpponentBoard, switchPlayer
+from utils import isValid
 
-from Board import initBoard, getBoardFromConfig, displayBoard
+# read configuration files from both players
+player1ConfigFile = sys.argv[1].split("=")[1]
+player2ConfigFile = sys.argv[2].split("=")[1]
 
-playerConfig = sys.argv[1].split("=")[1]
+with open("./config/" + player1ConfigFile, 'r') as f:
+    player1Config = f.read().splitlines()
 
-with open("./config/" + playerConfig, 'r') as f:
-    config = f.read().splitlines()
+with open("./config/" + player2ConfigFile, 'r') as f:
+    player2Config = f.read().splitlines()
 
-board = getBoardFromConfig(config)
-displayBoard(board)
+# initialize the boards
+boards["player1"]["primary"] = getBoardFromConfig(player1Config)
+boards["player1"]["opponent"] = initBoard(10)
+
+boards["player2"]["primary"] = getBoardFromConfig(player2Config)
+boards["player2"]["opponent"] = initBoard(10)
+
+# pick the first player randomly
+boards["currentPlayer"] = randint(1,2)
+
+while True:
+    if (boards["currentPlayer"] == 1 and boards["hit"] == 0):
+        os.system('clear')
+        input("PLAYER 2, look away. PLAYER 1, press enter when ready")
+        os.system('clear')
+        displayBoard(boards["player1"]["opponent"], "PLAYER 1'S TURN")
+        # displayBoard(boards["player1"]["primary"], "PLAYER 1")
+    elif boards["currentPlayer"] == 2 and boards["hit"] == 0:
+        os.system('clear')
+        input("PLAYER 1, look away. PLAYER 2, press enter when ready")
+        os.system('clear')
+        displayBoard(boards["player2"]["opponent"], "PLAYER 2'S TURN")
+
+
+    while True:
```

Comment:
You should not use a `while true` loop here imo. 
Maybe :
`while not isValid(coord)`